### DISCUSSION TITLE
How can we define the expected type for a JSONB column

### DISCUSSION THREAD

**Discussion by vladfrangu** [prisma/prisma#5254] [Q&A]
Hello!

I'm currently in the process of migrating my project to prisma, but I've hit one confusing roadblock..

As it stands, I have a `JSONB` column with a known format that I'd like to have strictly typed, however that is not possible as far as I know..

What would be the preferred way of me telling the generated TS types that *this* is the actual type for it?

**Comment by ryands17** [comment]
Hey @vladfrangu 👋 

Typing a `Json` column is currently not straightforward in Prisma as you would need to explicitly cast the type after you receive it in the response.

There was also a discussion previously about this https://github.com/prisma/prisma/discussions/3894 and there's also a feature request for the same which you can follow for more updates.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
